stable_baselines3/common/env_checker.py

```diff
@@ -169,6 +169,41 @@ def _check_obs(obs: Union[tuple, dict, np.ndarray, int], observation_space: spac
     elif _is_numpy_array_space(observation_space):
         assert isinstance(obs, np.ndarray), f"The observation returned by `{method_name}()` method must be a numpy array"
 
+    if isinstance(observation_space, spaces.Box):
+        # check obs dimensions, dtype and bounds
+        assert (
+            observation_space.shape == obs.shape
+        ), """\
```

Comment:
Please use f-strings, they also work with multi lines strings, or do something like:
```python
assert ..., (f"long message with {var1}" 
             f"rest of the message with {var2}")
```

(note: black will reformat it the correct way)